tests/ui/conftest.py

```diff
@@ -9,11 +9,21 @@
 from django.core.management import call_command
 from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
 from olympia.amo.tests import create_switch
+from selenium.webdriver.common.action_chains import ActionChains
+
 from pytest_django import live_server_helper
 
 
+@pytest.fixture
+def selenium(selenium):
+    selenium.implicitly_wait(10)
```

Comment:
We don't want to mix implicit and explicit waits. Please remove this and make sure you use explicit waits wherever they're necessary.